### ISSUE TITLE
Helper: Sounds to define up/down tree

### ISSUE THREAD

**Issue by willwade** [AceCentre/pasco#245]
Have a auditory cue to denote where you are in the tree..

E.g.

Quick Words (On pressing -  2 plays of https://soundbible.com/819-Checkout-Scanner-Beep.html) 
    Back (On pressing play once of https://soundbible.com/819-Checkout-Scanner-Beep.html) 
    Hi
    Whats up?
    Are you ok?
    I want something (On pressing -  3 plays of https://soundbible.com/819-Checkout-Scanner-Beep.html) 
          Back (On pressing -  2 plays of https://soundbible.com/819-Checkout-Scanner-Beep.html) 
          Help
          Food
          Drinks

Ideally have the audio for down and up a tree configurable.. (Select a audio file or record it)

**Comment by gavinhenderson** [comment]
@willwade Could we get away with doing something like this using current meta tags? Over ride the cue prompt to speak out what level you are currently at (followed by the actual cue)? Like this:

```
    Back <meta data-auditory-cue="Level One Back">
    Hi
    Whats up?
    Are you ok?
    I want something <meta data-auditory-cue="Level Two I want something">
          Back <meta data-auditory-cue="Level Three Back">
          Help
          Food
          Drinks
```

**Comment by willwade** [comment]
Yeah we could do that. We could just do with making an example or documenting it

**Comment by gavinhenderson** [comment]
Gotcha, ill keep this issue but switch it to add documentation / example
